lib/note-list/no-notes.tsx

```diff
@@ -29,15 +28,60 @@ const NoNotes = () => {
       </button>
     ) : (
       <button onClick={() => dispatch(actions.ui.createNote())}>
-        Create a new note
+        Create your first note
       </button>
     );
   };
 
+  type EmptyNoteListPlaceholder = {
+    message: String;
+    icon: JSX.Element | null;
+    button: JSX.Element | null;
+  };
```

Comment:
let's keep our type definitions in the module-global scope and towards the top of the module, under the imports

that will help avoid "hunting" for them in the code

Reply:
Thanks! I moved that now.